unittests/CMakeLists.txt

```diff
@@ -81,7 +81,8 @@ target_include_directories( unit_test PUBLIC
                             ${CMAKE_CURRENT_SOURCE_DIR}/contracts
                             ${CMAKE_CURRENT_BINARY_DIR}/contracts
                             ${CMAKE_CURRENT_BINARY_DIR}/include
-                            ${CMAKE_SOURCE_DIR}/plugins/http_plugin/include )
+                            ${CMAKE_SOURCE_DIR}/plugins/http_plugin/include
+                            ${CMAKE_SOURCE_DIR}/plugins/chain_interface/include)
```

Comment:
Not part of your work, but `unittests/plugin_tests.cpp` should be moved to `tests` so these are not needed. I added a comment here: https://github.com/AntelopeIO/leap/issues/586#issuecomment-1440273072